source/renderer/app/containers/MainLayout.js

```diff
@@ -21,15 +21,15 @@ export default class MainLayout extends Component<InjectedContainerProps> {
 
   render() {
     const { actions, stores } = this.props;
-    const { sidebar, wallets, profile, app } = stores;
-    const activeWallet = wallets.active;
+    const { sidebar, profile, app } = stores;
+    const activeWallet = stores.wallets.active;
```

Comment:
@DeeJayElly you can declare wallets store inside `const { sidebar, profile, app } = stores;`

Reply:
Fixed